docs/architecture/microservices/multi-container-microservice-net-applications/implement-api-gateways-with-ocelot.md

```diff
@@ -192,7 +192,7 @@ The important point here for Ocelot is the `configuration.json` file that you mu
 
 There are two sections to the configuration. An array of Re-Routes and a GlobalConfiguration. The Re-Routes are the objects that tell Ocelot how to treat an upstream request. The Global configuration allows overrides of Re-Route specific settings. It’s useful if you don’t want to manage lots of Re-Route specific settings.
 
-Here’s a simplified example of [ReRoute configuration file](https://github.com/dotnet-architecture/eShopOnContainers/blob/dev/src/ApiGateways/Web.Bff.Shopping/apigw/configuration.json) from one of the API Gateways from eShopOnContainers.
+Here’s a simplified example of [ReRoute configuration file](https://github.com/dotnet-architecture/eShopOnContainers/blob/master/src/ApiGateways/Web.Bff.Shopping/apigw/configuration.json) from one of the API Gateways from eShopOnContainers.
```

Comment:
The `dev` branch is the default one for that repository. If that file was deleted from that branch, I think there should be a reason for that. Hence, the article may be needed to get updated in a different way.

Reply:
We should check with @nishanil or @CESARDELATORRE on that question.

Reply:
We've dropped the support for Ocelot in eShop code but will retain the documentation in the book. As part of 3.1 updates (https://github.com/dotnet/docs/pull/16202), this guide will be updated to reflect changes.  For now, the redirection to master is correct.
@mvelosop  Can you note this and update the link in your pr?